sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewViewModel.groovy

```diff
@@ -29,12 +30,14 @@ class ProjectOverviewViewModel {
     @Bindable String generatedManifest
     final Subscriber subscriber
     final Channel<String> updatedProjectsChannel
+    final Map<String,CheckBox> projectToCheckbox = new HashMap<>()
```

Comment:
This should be in the factory in my opinion so that the logic of whether a new object is created or an old one is reused is contained there.

Reply:
Ja, I see your point here. I was also unhappy to have a view component in the model